sass/custom/_exec.scss

```diff
@@ -1,24 +1,49 @@
+.role {
+    display: flex;
+    align-items: center;
+}
+
+.role>* {
+    margin: 1vh 0;
+}
+
+.role a {
+    margin-left: 0.5vw;
+}
+
+.exec-title img {
+    margin: 0 1vw;
+    transform: scale(1.5);
+}
+
+.exec-title h3 {
+    margin: 0;
+}
+
 @include media-breakpoint-down(sm) {
-    .exec-table thead {
-        display: none;
+    .role {
+        display: flex;
+        justify-content: space-between;
     }
 
-    .exec-table tr>* {
-        display: block;
-        border-color: var(--bs-table-border-color);
+    .exec-title img {
+        transform: scale(1);
     }
 
-    .exec-table tr {
-        display: block;
-        border-bottom: none;
-        margin-top: 4vh;
+    .exec-title h3 {
+        font-size: 1rem;
+        margin-left: 1vw;
     }
+}
 
-    .exec-table tr>*:first-of-type {
-        border-top-width: var(--bs-border-width);
-    }
+.exec-cards {
+    width: 100%;
+    display: grid;
+    gap: 1rem;
+    grid-template-columns: repeat(auto-fill, minmax(150px, 1fr));
```

Comment:
rem size here

Reply:
this is the standard way of doing grid with a fr donating the main size and px for the limit, its the standatd for a reason